src/ceph-volume/ceph_volume/tests/api/test_lvm.py

```diff
@@ -143,7 +143,7 @@ def test_bit_more_than_half_size_allocates_full_size(self):
         assert result['percentages'] == 100
         assert result['parts'] == 1
 
-    def test_extents_are_halfed_rounded_down(self):
+    def test_extents_are_halved_rounded_down(self):
```

Comment:
Are calls updated to match?

Reply:
This appears to be a pytest which is an automatically called thing (it takes some time to get used to this behavior). https://cs.github.com/?scopeName=All+repos&scope=&q=repo%3Aceph%2Fceph+test_coment_as_a_value shows that it's the only instance of that token in the repository. w/o the repo filter, there are only 2 other instances which are just the same file in forks.